improver_tests/acceptance/SHA256SUMS

```diff
@@ -577,18 +577,18 @@ eff5c671c33c3520e9e7f0bcab4fe2a4eb5557fd84767da1a8c494f3ab7f3aa6  ./spot-extract
 522f5331735a778f65ad5ed6ca5ebca03bdcb8d4c7a783d6f7afe61b61efa0ab  ./spot-extract/outputs/nearest_uk_temperatures_amended_metadata.nc
 7f7734eb79abe416b0587cff2fbff286854e978037be3d63054c93d67b8bdc39  ./spot-extract/outputs/nearest_uk_temperatures_unique_ids.nc
 bf26c1be428a207a20d3a2b54987b24435944aaf52135e297176cc11f3fc3b97  ./spot-extract/outputs/with_realization_collapse.nc
-b3080315914473c89b814adae8a6e7306afb42248212177702acc86b0f5878ae  ./standardise/float64/float64_data.nc
-b40913e8086865f3a0fdcc6793a67dfe695710169c40d3e23f120920eceed0ba  ./standardise/float64/kgo.nc
-9c834da03db5f25627c1be3ff666718ae5186c77895854e3d84db8ce56af16cf  ./standardise/metadata/input.nc
-ac3ded21e9d3f3aead4288101ea574e019772e9479534910b616d0d38d45f8d7  ./standardise/metadata/kgo.nc
+b9fca4fb2a70c9d1eb49cc032fd298cf75a6c68997f419ffe62db067775ad437  ./standardise/float64/float64_data.nc
+4265519a1c74f45f1eb14f2a5b164e92e0744ac1aa670c04c9bac2b6c4b67e77  ./standardise/float64/kgo.nc
+867c0c8944c8c97b3f950c699b1fc78fb16fea1d0c0767e948ab07ad1d043d6f  ./standardise/metadata/input.nc
+fb95006945ef65aba58b6464cbe789439c6c2fce982629e502b65483e31b3702  ./standardise/metadata/kgo.nc
 d4404df3a8acdeed27f20ec621a0d3500bf4c849eaf7a7064c4e42f079b37a43  ./standardise/metadata/metadata.json
 326ae7b6d3cff0fd3da840b643816d897e7133a248e49eacad3ec37af669f49e  ./standardise/metadata/radar_metadata.json
 d89a8587bc28b574b8b1e624bb3bf339aaeb9c8c2355db7fe518af8c9bce527c  ./standardise/radarnet/input_coverage.nimrod
 6335cba81be74577fe10d0a4f5cb75724abcef16499d76c263e80c16cf092a05  ./standardise/radarnet/input_preciprate.nimrod
 8a0cc7c53513fe11919da6ae39e431fde7f5b5831e255d8f30feda2a53eb3c71  ./standardise/radarnet/kgo_coverage.nc
 8e4ca14de258f9a511c9ff82ac7df5cc95659bc41c3e745083f5ed449f245d00  ./standardise/radarnet/kgo_preciprate.nc
-4a6d679776a8f7ac35dff33093e1558a2b497527d1537b6bdc5d325bca29097b  ./standardise/stage-v110/input.nc
-575151096829ed458ab13702129efc88f48a8ec459f9b3f39e11e4f5b2e296b2  ./standardise/stage-v110/kgo.nc
+44cf0fd7d6a4d24546da814d08f87f58d2f83ef5ae3a94c4272a4d33659c3f24  ./standardise/stage-v110/input.nc
+add67f9b08aade71cac803044d650a966698820b8e889468b760edaf3bf89dcc  ./standardise/stage-v110/kgo.nc
```

Comment:
This test is obsolete. Can you delete it please.

Reply:
If we want to delete this test, we should remove the functionality that it is testing. I've created an issue to handle this separately to this data volume reduction exercise: https://github.com/metoppv/mo-blue-team/issues/457

Reply:
That's sensible.